src/components/footer/index.tsx

```diff
@@ -131,17 +140,20 @@ function Footer({
 						</li>
 					)
 				})}
-				<li>
+				<li className={classNames(s.actionDesktop, s.linkListItem)}>
 					<ButtonLink
 						text="Give Feedback"
 						href={FEEDBACK_FORM_URL}
 						color="secondary"
 						size="small"
-						icon={<IconPencilTool16 />}
 						opensInNewTab={true}
-						className={classNames(s.feedbackButton, s.desktop)}
 					/>
 				</li>
+				{shouldRenderThemeSwitcher ? (
+					<li className={classNames(s.actionDesktop, s.linkListItem)}>
+						<ThemeSelect ariaLabel="Select a theme" />
```

Comment:
🚧 Blocking: With this value for `aria-label`, the `<select>`'s Accessible Name becomes "Select a theme". What that means for voice control users, is that they have to speak "Select a theme" in order to activate the element. Since there's no visible label with this text (hence why `aria-label` was reached for), voice control users aren't going to be able to activate this element except by pure luck and guessing that text.

🙋🏻 A few questions for my own context gathering:

- Did we explore adding a visible label for this switcher?
- Did we explore placing a label "inside" the switcher, maybe something like `Theme: ${option.label}`?

Reply:
@ashleemboyer is there a way to communicate the label to voice control users without visibly showing the label?

Reply:
Additional context will help me make a recommendation here 😊

Reply:
<img width="1458" alt="Screenshot 2023-04-04 at 11 39 55 AM" src="https://user-images.githubusercontent.com/36613477/229887941-21853175-2dcd-4520-bdad-cf6e8589c665.png">
We use the same approach on our install version switcher

Reply:
We wanted to be lighter on the footer treatment as:

1. The footer is the secondary place to change theme (need to validate this hypothesis)
2. There is a ton of content in the footer, adding a label creates some complications with the footer designs especially when we get into the smaller viewports.

Reply:
<img width="1250" alt="Screenshot 2023-04-04 at 12 12 59 PM" src="https://user-images.githubusercontent.com/36613477/229896383-421af8b6-3827-4811-b8cb-d81152608059.png">
@ashleemboyer could you share why its okay to not have a visual label here but we need to refactor to add one in this case?

Reply:
This feedback wasn't in the initial design review and it feels a late to fully change up the designs to support this. Could we potentially merge this iteration in and then follow up to address the label issue when we have space to explore the right design solution.

Reply:
@ashleemboyer I updated the `aria-label` to read `theme` and I added a ticket in this aria label improvements task here to follow up on this. https://app.asana.com/0/1202097197789424/1204334637222649/f

Since this requires a design dependency, we would like to move forward with this implementation as is and address the above concerns separately in a design exercise.